plugins/constraints/package.json

```diff
@@ -47,14 +47,14 @@
   },
   "license": "ISC",
   "dependencies": {
-    "@flatfile/plugin-record-hook": "^1.4.7"
   },
   "peerDependencies": {
-    "@flatfile/api": "^1.7.4",
-    "@flatfile/listener": "^1.0.1"
+    "@flatfile/api": "^1.8.0",
+    "@flatfile/listener": "^1.0.1",
+    "@flatfile/plugin-record-hook": "^1.5.1"
   },
   "devDependencies": {
-    "@flatfile/api": "^1.7.4",
+    "@flatfile/plugin-record-hook": "^1.5.1",
```

Comment:
Shouldn't this only be in `peerDependencies`?
```suggestion
```

Reply:
Same as above.